apps/framework-docs/src/pages/getting-started/_meta.json

```diff
@@ -1,6 +1,7 @@
 {
-    "version-status": "Alpha warning",
-    "new-project": "Quick start",
-    "project-structure": "Project Structure",
-    "architecture": "Architecture"
-}
\ No newline at end of file
+  "version-status": "Alpha Warning",
+  "new-project": "Quick Start",
+  "project-structure": "Project Structure",
+  "primitives-overview": "Primitives Overview",
+  "architecture": "Architecture"
+}
```

Comment:
how did the previous formatting get pass CI🧐

Reply:
something strange is happening, Primitives Overview shows in sidebar locally but preview deployment does not display

Reply:
That's a cache issue - doing a redeploy without cache should fix it